### ISSUE TITLE
False positive on parallel assignment

### ISSUE THREAD

**Issue by ojab** [YorickPeterse/ruby-lint#147]
With code
```ruby
xxx = { a: 1, b: 2 }
xxx[:a], xxx[:b] = 1, 2
```
ruby-lint 2.0.4 reports
```
$ ruby-lint 1.rb
1.rb: error: line 2, column 1: wrong number of arguments (expected 2 but got 1)
1.rb: error: line 2, column 10: wrong number of arguments (expected 2 but got 1)
```
